lib/Backend/BackwardPass.cpp

```diff
@@ -1144,10 +1144,6 @@ BackwardPass::MergeSuccBlocksInfo(BasicBlock * block)
         }
     }
     block->upwardExposedUses = upwardExposedUses;
-    if (this->tag == Js::BackwardPhase)
-    {
-        block->successorBlockUses = upwardExposedUses ? upwardExposedUses->CopyNew(this->globOpt->alloc) : nullptr;
-    }
     block->upwardExposedFields = upwardExposedFields;
```

Comment:
I was using this bit vector to check if a dst opnd is used beyond this block, but it is conservation. It is sufficient to check upwardExposedUsed in the direction we can prove the branch.